crates/aptos-profiler/src/cpu_profiler.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use crate::{CpuProfilerConfig, Profiler};
+use crate::utils::{convert_svg_to_string, create_file_with_parents};
+use std::{thread, time, path::PathBuf};
+
+pub struct CpuProfiler {
+    duration: u64,
```

Comment:
nit: either use a Duration struct, or rename the variable with unit (e.g. duration_secs)